vendor/wp-now/src/wp-now.ts

```diff
@@ -563,3 +572,15 @@ export async function moveDatabasesInSitu( projectPath: string ) {
 		fs.rmSync( wpContentPath, { recursive: true, force: true } );
 	}
 }
+
+function addDefaultUmaskPlugin( php: PHP, documentRoot: string ) {
+	php.writeFile(
+		'/internal/shared/mu-plugins/override-default-umask.php',
```

Comment:
All plugins in `/internal/shared/mu-plugins` are loaded by Playground ([reference](https://github.com/WordPress/wordpress-playground/blob/a55dcc01dcb7ac62b6ea7b99949e01b01f4e92cd/packages/playground/wordpress/src/index.ts#L33-L47)). Placing internal plugins in this location will avoid including them when exporting the site.

Reply:
For reference, here's the issue: https://github.com/Automattic/dotcom-forge/issues/9235